sipservice/src/main/java/org/pjsip/pjsua2/extras/CallbackMessageConst.java

```diff
@@ -0,0 +1,5 @@
+package org.pjsip.pjsua2.extras;
+
+public class CallbackMessageConst {
+    public static final String ERR_SIP_ACCOUNT_NULL = "Sip account in not found.";
```

Comment:
Move this constant to SipServiceConstant

Reply:
All changes will be accommodated in AND-3730 PR.